src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -315,3 +317,22 @@ def prepare_logs_argument(c):
                   'spring-cloud application-configuration-service git repo remove']:
         with self.argument_context(scope) as c:
             c.argument('name', help="Required unique name to label each item of git configs.")
+
+    for scope in ['spring-cloud build-service buildpacks-binding create',
+                  'spring-cloud build-service buildpacks-binding set']:
```

Comment:
update? Usually use pairs

[create, update, show, delete]
[add, remove]

Reply:
update may be partial, use `set` here to emphasize replace here.